templates/shared/forms/_country_list.html

```diff
@@ -0,0 +1,271 @@
+<label for="country"
```

Comment:
Sorry, why is a new one needed? This looks very similar to https://github.com/canonical/canonical.com/blob/main/templates/shared/forms/_country.html , shouldn't we just make whatever changes are needed there rather than having two files that seemingly do the same thing?

Reply:
As I mentioned above, I added a new countries template, so that we can slowly decomission the old one, without breaking the other forms.

Reply:
This is the problem if I use the `_country.html` 
![Screenshot from 2024-05-23 16-20-18](https://github.com/canonical/canonical.com/assets/14939793/42f650fe-d362-41d7-9d0b-862e32709de0)

Reply:
I understand the problem, my point is that it seems unnecessary to repeat so much code when adding a small bit of jinja logic to the original file (as you've already done) seems to address this issue

Reply:
Ok. I've refactored it with a condition.